src/main/java/com/es/phoneshop/security/CustomDosProtectionService.java

```diff
@@ -1,15 +1,18 @@
 package com.es.phoneshop.security;
 
+import lombok.Getter;
+
 import java.util.Map;
 import java.util.concurrent.ConcurrentHashMap;
 
+@Getter
 public class CustomDosProtectionService implements DosProtectionService {
 
-    private static final long TIME_INTERVAL_MILLS = 60000;
+    private static final long TIME_INTERVAL_MILLS = 60_000;
```

Comment:
why 60_000

Reply:
this is for clarity. "_" not taken into account when compiling